controllers/ingress_controller.go

```diff
@@ -121,13 +121,15 @@ func (r *IngressReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 }
 
 func (r *IngressReconciler) keyless() *v1alpha1.ApiDefinition {
+	useKeyless := true
+
 	return &v1alpha1.ApiDefinition{
 		Spec: v1alpha1.APIDefinitionSpec{
 			APIDefinitionSpec: model.APIDefinitionSpec{
 				Name:             "default-keyless",
 				Protocol:         "http",
-				UseKeylessAccess: true,
-				Active:           true,
+				UseKeylessAccess: &useKeyless,
```

Comment:
Is it safe to use same variable's address for two fields? I mean, if `UseKeylessAccess`'s value is changed to false, it'll affect `Active` as well

Reply:
Opps sorry. 
Didn't intend to do it that way. Will fix it